install/installation-macos.md

```diff
@@ -93,60 +98,41 @@ installed it yet, check out our [installing psql][install-psql] section.
     If your connection is successful, you'll see a message like this, followed
     by the `psql` prompt:
     ```
-    psql (13.3, server 12.8 (Ubuntu 12.8-1.pgdg21.04+1))
-    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+    psql (14.4)
     Type "help" for help.
-    tsdb=>
+
     ```
-1.  At the `psql` prompt, create an empty database. Our database is
-    called `example`:
+1.  At the `psql` prompt, create an empty database named `tsdb`:
     ```sql
-    CREATE database example;
+    CREATE database tsdb;
     ```
-1.  Connect to the database you created:
+1.  Connect to the `tsdb` database that you created:
     ```sql
-    \c example
+    \c tsdb
     ```
 1.  Add the TimescaleDB extension:
     ```sql
     CREATE EXTENSION IF NOT EXISTS timescaledb;
     ```
-1.  You can now connect to your database using this command:
-    ```bash
-    psql -U postgres -h localhost -d example
-    ```
-
-</procedure>
-
-You can check that the TimescaleDB extension is installed by using the `\dx`
-command at the `psql` prompt. It looks like this:
+1. Check that the TimescaleDB extension is installed by using the `\dx`
+command at the `psql` prompt. Output is similar to:
 ```sql
-tsdb=> \dx
-List of installed extensions
--[ RECORD 1 ]------------------------------------------------------------------
-Name        | pg_stat_statements
-Version     | 1.7
-Schema      | public
-Description | track execution statistics of all SQL statements executed
--[ RECORD 2 ]------------------------------------------------------------------
-Name        | plpgsql
-Version     | 1.0
-Schema      | pg_catalog
-Description | PL/pgSQL procedural language
--[ RECORD 3 ]------------------------------------------------------------------
-Name        | timescaledb
-Version     | 2.5.1
-Schema      | public
-Description | Enables scalable inserts and complex queries for time-series data
--[ RECORD 4 ]------------------------------------------------------------------
-Name        | timescaledb_toolkit
-Version     | 1.3.1
-Schema      | public
-Description | timescaledb_toolkit
-
-tsdb=>
+tsdb-# \dx
+                                      List of installed extensions
+    Name     | Version |   Schema   |                            Description                            
+-------------+---------+------------+-------------------------------------------------------------------
+ plpgsql     | 1.0     | pg_catalog | PL/pgSQL procedural language
+ timescaledb | 2.7.0   | public     | Enables scalable inserts and complex queries for time-series data
+(2 rows)
 ```
```

Comment:
This needs to be indented, I think. It's tabbed too far left in the output:

![image](https://user-images.githubusercontent.com/3914967/174507111-da94b98d-bc27-45cc-80ae-bf9a51086ccf.png)

Reply:
Thanks @Loquacity !!! Just fixed the issue.